hack/make-rules/verify.sh

```diff
@@ -21,15 +21,10 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-if [ -n "${VERBOSE}" ]; then
```

Comment:
I've noticed that, but by default when running `make verify` verbose is set. Should I remove them from there?